js/app.js

```diff
@@ -0,0 +1,89 @@
+document.addEventListener(`DOMContentLoaded`, () =>{
```

Comment:
Teoretycznie tutaj tego nie potrzebujesz. Dodałeś `script` na samym końcu `body` więc zostanie załadowany dopiero kiedy załaduje się strona

Reply:
było w cwiczeniu, zostawiłem tez po to, żeby sobie zapamietać, ale bylem swiadom, ze jest zbedne.